src/repeat-prevent.coffee

```diff
@@ -1,7 +1,39 @@
 # Description:
-#   A hubot script to do nothing
+#   A hubot script to prevent responses if they repeat a recent response
 #
 # Author:
 #   Andrew Lucas (sqweelygig) <andrewl@resin.io> <sqweelygig@gmail.com>
+moment = require 'moment'
+_ = require 'lodash'
+
+scopes = {}
```

Comment:
Naming nit: What does `scopes` contain? Confusingly, not scopes - those are the keys. Could be `messageTimesByScope` or something?